src/core/Song.cpp

```diff
@@ -1054,6 +1054,23 @@ void Song::loadProject( const QString & fileName )
 	}
 
 	node = dataFile.content().firstChild();
+
+	QDomNodeList tclist=dataFile.content().elementsByTagName("trackcontainer");
+	m_nLoadingTrack=0;
+	for( int i=0,n=tclist.count(); i<n; ++i ){
+		QDomNode nd=tclist.at(i).firstChild();
+		while(!nd.isNull()){
```

Comment:
I think the general idea is to put the braces on the same column/tab although I'm not 100% sure about the convention around this.

```

( .. )
{
  this
}
```


```

( .. ){
  not this
}
```